daemon/inertiad/project/deployment.go

```diff
@@ -314,6 +314,32 @@ func (d *Deployment) CompareRemotes(remoteURL string) error {
 	return nil
 }
 
+// UpdateContainerHistory will update container bucket with recent build's
+// metadata
+func (d *Deployment) UpdateContainerHistory(cli *docker.Client) error {
+	// Retrieve container ID for recently deployed project
+	ctx := context.Background()
+	var containerID string
+	containers, err := cli.ContainerList(ctx, types.ContainerListOptions{})
+	if err != nil {
+		return err
+	}
+	for _, container := range containers {
+		if container.Names[0] == d.project {
```

Comment:
i think we might have to use `container.Names` to indicate build versions (I believe names must be unique, so if a project needs multiple containers for history, they cant all have the same name). We should add a new field to the container metadata (I think I sent you a link a while back) and put the project name there isntead, and check for it here

Reply:
jk this is fine for now I think